src/java.base/share/classes/jdk/internal/foreign/abi/SharedUtils.java

```diff
@@ -389,7 +389,7 @@ public VarHandle varHandle() {
         }
     }
 
-    public static non-sealed class EmptyVaList implements VaList {
+    public static final class EmptyVaList implements VaList {
```

Comment:
Rationale: while this is not strictly needed, I'm more willing to accept it since we had to say something (e.g. non-sealed) anyway. So, if in this case saying `final` is a more apt description, I'm ok with changing it.